reference/clinic.v1.yaml

```diff
@@ -1273,6 +1273,132 @@ paths:
       tags:
         - Clinics
         - Internal
+  /v1/redox:
+    post:
+      summary: Redox EHR Endpoint
+      operationId: ProcessEHRMessage
+      responses:
+        '200':
+          description: OK
+      tags:
+        - Redox
+      description: An endpoint for incoming Redox messages
+      requestBody:
+        content:
+          application/json:
+            schema:
+              type: object
+              properties: {}
+  /v1/redox/verify:
+    post:
+      summary: Redox Verify Endpoint
+      operationId: VerifyEndpoint
+      responses:
+        '200':
+          description: OK
+      tags:
+        - Redox
+        - Internal
+      x-internal: true
+      description: Internal endpoint for Redox chanallenge verification
+  '/v1/clinics/{clinicId}/settings/ehr':
```

Comment:
These will require `serverToken`?

Reply:
yes

Reply:
@krystophv requested making EHR and MRN settings accessible to clinic members for display purposes in blip